src/LibraryMangement/Home.js

```diff
@@ -0,0 +1,67 @@
+import React, { useState } from 'react'
+
+const Home = () => {
+
+    const BookName = [{ name: "science" }]
+    const [Booklist, setBookList] = useState(BookName);
```

Comment:
why we need to maintain both states `book` and `bookList` ? does a single does the job ?